File: 3432-count-partitions-with-even-sum-difference/3432-count-partitions-with-even-sum-difference.go
func countPartitions(nums []int) int {
var ele int = nums[0]
count:=0
for  i:=1;i<len(nums);i++ {
 var val int =find(i,nums)
  diff:=val-ele
  if diff%2==0{
    count++
  }
  ele+=nums[i]
  
}  
return count  
}
func find(n int ,nums[] int)int{
    sum:=0
    for i:=n;i<len(nums);i++{
        sum+=nums[i];
    }
    return sum
}

File: 3432-count-partitions-with-even-sum-difference/3432-count-partitions-with-even-sum-difference.java
class Solution {
    public int countPartitions(int[] nums) {
        int ele=nums[0];
        int count=0;
        for(int i=1;i<nums.length;i++)
        {
            int val=find(i,nums);
            int dif=ele-val;
            if(dif%2==0)
            {
                count++;
            }
            ele+=nums[i];
        }
        return count;
    }
    public static int find(int n,int nums[])
    {
        int sum=0;
        for(int i=n;i<nums.length;i++)
        {
        sum+=nums[i];
        }
        return sum;
    }
}